src/views/ipv/all-templates.njk

```diff
@@ -0,0 +1,24 @@
+{% extends "shared/base.njk" %}
+{% set pageTitleName = "All templates" %}
+
+{% block content %}
+  <table class="govuk-table">
+    <caption class="govuk-table__caption govuk-table__caption--l"></caption>
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">List of all pages in di-ipv-core-front</th>
+        <th scope="col" class="govuk-table__header"></th>
```

Comment:
Is it worth having this as a larger header than the content rows below? (Know that this isn't user facing, but might be better for readability nonetheless